src/5e-SRD-Features.json

```diff
@@ -206,22 +332,30 @@
       "Starting when you choose this path at 3rd level, you can go into a frenzy when you rage. If you do so, for the duration of your rage you can make a single melee weapon attack as a bonus action on each of your turns after this one. When your rage ends, you suffer one level of exhaustion (as described in appendix A)."
     ],
     "feature_specific": {
-      "bonus_action": {
-        "type": "attack",
-        "amount": 1
-      },
-      "prerequisites": [
-        {
-          "type": "boolean",
-          "from": "raging"
-        }
-      ],
-      "debuffs": [
+      "actions": [
         {
-          "index": "exhaustion",
-          "name": "Exhaustion",
-          "url": "/api/conditions/exhaustion",
-          "level": 1
+          "name": "Frenzy",
+          "desc": "When you enter rage, you can go into frenzy instead",
```

Comment:
Use the description from the SRD:

> You go	into	a	frenzy	when	you	rage.	For	
the	duration	of	your	rage	you	can	make	a	single	
melee	weapon	attack	as	a	bonus	action	on	each	of	
your	turns	after	this	one.	When	your	rage	ends,	you	
suffer	one	level	of	exhaustion	(as	described	in	
appendix	PH-A).go	into	a	frenzy	when	you	rage.	If	you	do	so,	for	
the	duration	of	your	rage	you	can	make	a	single	
melee	weapon	attack	as	a	bonus	action	on	each	of	
your	turns	after	this	one.	When	your	rage	ends,	you	
suffer	one	level	of	exhaustion	(as	described	in	
appendix	PH-A).

Reply:
ok. I've also split it into an array